cmd/restic/cmd_backup.go

```diff
@@ -270,6 +280,12 @@ func (opts BackupOptions) Check(gopts GlobalOptions, args []string) error {
 		if len(opts.FilesFromRaw) > 0 {
 			return errors.Fatal("--stdin and --files-from-raw cannot be used together")
 		}
+		if len(opts.Paths) > 0 {
+			return errors.Fatal("--stdin and --set-path cannot be used together")
+		}
+		if len(opts.PathsFrom) > 0 {
+			return errors.Fatal("--stdin and --set-paths-from cannot be used together")
+		}
```

Comment:
This is at conflict with the deprecation of the `--stdin-filename` option at [R120](https://github.com/restic/restic/pull/3200/files#diff-30d8672eafa9c0585766b6f437530377563cd1aac5606674c366f3ede4abea9eR120).

If ``--set-path``/``--set-paths-from`` is supposed to replace ``--stdin-filename``, then using ``--stdin`` together with them must be possible.